example/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
```

Comment:
It seems to have nothing to do with the function modification, should you delete it?